File: Chapter 6/Exercise/2. String.java
/*
 *  Name Program    : String
 *  Purpose         : Answer question from exercise Chapter 6 from Logic Algorithm and Basic Programming Book
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : Java Language
 */

import java.util.Scanner;

public class Main{

    // Main function
    public static void main(String[] args){

        // Variable declaration
        String[] input = new String[5];

        Scanner sc = new Scanner(System.in);

        System.out.println("Masukan lima buah karakter satu persatu, tekan enter setiap selesai memasukkan");

        // Get input from console
        /* 
            Example
            aku
            adalah
            anak
            gembala
            selalu
        */
        try{
            
            input[0] = sc.next();
            input[1] = sc.next();
            input[2] = sc.next();
            input[3] = sc.next();
            input[4] = sc.next();
        
        }catch(Exception e){

        }

        // Print to console
        System.out.println("Kata ke 1 = " + input[0]);
        System.out.println("Kata ke 2 = " + input[1]);
        System.out.println("Kata ke 3 = " + input[2]);
        System.out.println("Kata ke 4 = " + input[3]);
        System.out.println("Kata ke 5 = " + input[4]);
    }
}

File: Chapter 6/Exercise/2. String.c
/*
 *  Name Program    : String
 *  Purpose         : Answer question from exercise Chapter 6 from Logic Algorithm and Basic Programming Book
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : C Language
 */

#include <stdio.h>

// Main function
int main(){

    // Variable declaration
    char input[5][99];

    printf("Masukan lima buah kata secara berurutan, lalu tekan enter setelahnya\n");

    // Get input from console
    /* 
        Example
        aku
        adalah
        anak
        gembala
        selalu
     */
    scanf("%s", &input[0]);
    scanf("%s", &input[1]);
    scanf("%s", &input[2]);
    scanf("%s", &input[3]);
    scanf("%s", &input[4]);

    // Print to console
    printf("Kata ke - 1 = %s\n", input[0]);
    printf("Kata ke - 2 = %s\n", input[1]);
    printf("Kata ke - 3 = %s\n", input[2]);
    printf("Kata ke - 4 = %s\n", input[3]);
    printf("Kata ke - 5 = %s\n", input[4]);

    return 0;
}